lib/snapshot/libraries.yaml

```diff
@@ -2,158 +2,31 @@
-        - "../../../third_party/dart/sdk/lib/_internal/vm/lib/ffi_native_type_patch.dart"
-        - "../../../third_party/dart/sdk/lib/_internal/vm/lib/ffi_struct_patch.dart"
+    profiler:
+      uri: "../../../third_party/dart/sdk/lib/profiler/profiler.dart"
 
     wasm:
       uri: "../../../third_party/dart/sdk/lib/wasm/wasm.dart"
       patches:
         - "../../../third_party/dart/sdk/lib/_internal/vm/lib/wasm_patch.dart"
 
-    _http:
-      uri: "../../../third_party/dart/sdk/lib/_http/http.dart"
-
-    io:
-      uri: "../../../third_party/dart/sdk/lib/io/io.dart"
-      patches:
-        - "../../../third_party/dart/sdk/lib/_internal/vm/bin/common_patch.dart"
-        - "../../../third_party/dart/sdk/lib/_internal/vm/bin/directory_patch.dart"
-        - "../../../third_party/dart/sdk/lib/_internal/vm/bin/eventhandler_patch.dart"
-        - "../../../third_party/dart/sdk/lib/_internal/vm/bin/file_patch.dart"
-        - "../../../third_party/dart/sdk/lib/_internal/vm/bin/file_system_entity_patch.dart"
-        - "../../../third_party/dart/sdk/lib/_internal/vm/bin/filter_patch.dart"
-        - "../../../third_party/dart/sdk/lib/_internal/vm/bin/io_service_patch.dart"
-        - "../../../third_party/dart/sdk/lib/_internal/vm/bin/namespace_patch.dart"
-        - "../../../third_party/dart/sdk/lib/_internal/vm/bin/platform_patch.dart"
-        - "../../../third_party/dart/sdk/lib/_internal/vm/bin/process_patch.dart"
-        - "../../../third_party/dart/sdk/lib/_internal/vm/bin/socket_patch.dart"
-        - "../../../third_party/dart/sdk/lib/_internal/vm/bin/stdio_patch.dart"
-        - "../../../third_party/dart/sdk/lib/_internal/vm/bin/secure_socket_patch.dart"
-        - "../../../third_party/dart/sdk/lib/_internal/vm/bin/sync_socket_patch.dart"
-
-    isolate:
-      uri: "../../../third_party/dart/sdk/lib/isolate/isolate.dart"
-      patches:
-        - "../../../third_party/dart/sdk/lib/_internal/vm/lib/isolate_patch.dart"
-        - "../../../third_party/dart/sdk/lib/_internal/vm/lib/timer_impl.dart"
-
-    math:
-      uri: "../../../third_party/dart/sdk/lib/math/math.dart"
-      patches: "../../../third_party/dart/sdk/lib/_internal/vm/lib/math_patch.dart"
-
-    mirrors:
-      uri: "../../../third_party/dart/sdk/lib/mirrors/mirrors.dart"
-      patches:
-        - "../../../third_party/dart/sdk/lib/_internal/vm/lib/mirrors_patch.dart"
-        - "../../../third_party/dart/sdk/lib/_internal/vm/lib/mirrors_impl.dart"
-        - "../../../third_party/dart/sdk/lib/_internal/vm/lib/mirror_reference.dart"
-
-    nativewrappers:
-      uri: "../../../third_party/dart/sdk/lib/html/dartium/nativewrappers.dart"
-
-    profiler:
-      uri: "../../../third_party/dart/sdk/lib/profiler/profiler.dart"
-
-    typed_data:
-      uri: "../../../third_party/dart/sdk/lib/typed_data/typed_data.dart"
-      patches: "../../../third_party/dart/sdk/lib/_internal/vm/lib/typed_data_patch.dart"
+    _spirv:
```

Comment:
So this was wrongfully missing from the .yaml version before, right?

Reply:
Yes. libraries.json for Flutter and Flutter runner include "_spirv" library but corresponding libraries.yaml don't. As far as I understand CFE uses libraries.json, so "_spirv" library is in fact included into the core libraries. So I updated libraries.yaml to reflect that and re-generated libraries.json.

Reply:
_spirv library should have been deleted when we removed the transpiler. It's possible we missed something though

Reply:
Yes, I believe all of the _spirv imports are removed. Fyi @zanderso

Reply:
I checked other Flutter core libraries and found that in addition to '_spirv' there are also obsolete 'wasm' and 'profiler' libraries which reference non-existing .dart files. Cleaned up all of them, PTAL.

Reply:
Correct. This reminded me to do this https://github.com/flutter/engine/pull/36422...